en/docs/concepts/identity-provisioning-intro.md

```diff
@@ -0,0 +1,43 @@
+# Introduction to Identity Provisioning
+
+## What is identity provisioning? 
+
+Identity provisioning is the process of creating, updating, managing, and removing digital identities in a system or application. The main purpose of identity provisioning using an identity management system like WSO2 Identity Server (WSO2 IS), is to automate the process of creating and managing these user identities so that it can be done as efficiently as possible, even on multiple platforms or applications. 
+
+----
+
+## Why do I need provisioning?
+
+Most enterprise solutions adopt products and services from multiple cloud providers to accomplish various business requirements. This makes it insufficient to maintain user identities only in a corporate LDAP. Managing user accounts on multiple applications can be challenging and can lead to outdated information, outdated permissions, data mismatch, and security issues. Identity provisioning plays a key role in propagating user identities across different SaaS providers.
```

Comment:
WE have to say that "due to this reason we have to maintain user information in multiple places"